packages/common-utils/src/types.ts

```diff
@@ -73,6 +73,7 @@ export const SelectListSchema = z.array(DerivedColumnSchema).or(z.string());
 export const SortSpecificationSchema = z.intersection(
   RootValueExpressionSchema,
   z.object({
+    valueExpression: z.string().optional(),
```

Comment:
why do we need to add this?

Reply:
It get mentioned in `lint` error,
usually ordering will need to mention column, example:
```
orderBy: [
        {
          valueExpression: config.timestampValueExpression,
          ordering: 'ASC',
        },
      ],
```

I am fine to revert this, because I don't think I need `order` query anymore(data need to sort after fetching anyway).

Reply:
fwiw the type should work since it's intersecting with the `RootValueExpressionSchema`